rhizome/host/bin/setup-slice

```diff
@@ -0,0 +1,42 @@
+#!/bin/env ruby
+# frozen_string_literal: true
+
+require_relative "../lib/slice_setup"
+
+unless (action = ARGV.shift)
+  puts "expected action as argument"
+  exit 1
+end
+
+unless (slice_name = ARGV.shift)
+  puts "expected slice name as argument"
+  exit 1
+end
+
+slice_setup = SliceSetup.new(slice_name)
+
+case action
+when "delete"
+  slice_setup.purge
+
+when "prep"
+  unless (allowed_cpus = ARGV.shift)
```

Comment:
maybe you should check the format of these parameters. I can see that you pass an `int-int` pair in the tests. Just in case, in future if the order of parameters are messed up, we can catch it here?

Reply:
I added a check in `SliceSetup.prep` to fail if format is not correct.